.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -1,6 +0,0 @@
-<component name="InspectionProjectProfileManager">
-  <profile version="1.0">
-    <option name="myName" value="Project Default" />
-    <inspection_tool class="TypeAnnotation" enabled="false" level="WARNING" enabled_by_default="false" />
-  </profile>
-</component>
```

Comment:
Was gonna comment on the non-newlined EOF. Then I realised this whole file is getting the boot, so that's all good!